basic_system/src/system_implementation/flat_storage_model/storage_cache.rs

```diff
@@ -124,30 +132,37 @@ where
             current_tx_number: TransactionId(0),
             resources_policy,
             initial_values: BTreeMap::new_in(allocator.clone()),
+            evm_refunds_counter: HistoryCounter::new(allocator.clone()),
             alloc: allocator.clone(),
             _marker: core::marker::PhantomData,
         }
     }
 
     pub fn begin_new_tx(&mut self) {
         self.cache.commit();
+        // TODO
```

Comment:
Can be removed

Reply:
Done

Reply:
I thought that maybe we can reset this counter in some better way and reduce amount of cycles spent